app/services/users/profile.rb

```diff
@@ -63,6 +65,22 @@ def account
       nil
     end
 
+    # Get the preferred named from the user's demographics. See scaffold.errors for any errors.
+    #
+    # @return the user's preferred name or nil if the user has no preferred name or there was an
+    # error fetching the name.
+    #
+    def demographics_data
+      demographic_svc = VAProfile::Demographics::Service.new @user
```

Comment:
Minor detail I overlooked previously, but we can change this to reference `user` instead of `@user` since we are using the `attr_reader` to define the getter method. Not a super big deal, but since the rest of the class uses `user` we might as well stay consistent.